mcarch/login.py

```diff
@@ -67,6 +67,9 @@ def log_in(uname, passwd):
     """Attempts to log in with the given credentials. Returns true on success, false on failure."""
     user = User.query.filter_by(name=uname).first()
     if not user: return False
+    if user.disabled:
+        flash("User disabled")
```

Comment:
This error message could allow an attacker to discover usernames of disabled users. Probably not a big deal, but it's best to avoid it.